component/rbac.libsonnet

```diff
@@ -0,0 +1,35 @@
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+local inv = kap.inventory();
+
+local metrics_role(name, namespace) =
+  kube.Role(name) {
+    metadata+: {
+      namespace: namespace,
+    },
+    rules: [
+      {
+        apiGroups: [ '' ],
+        resources: [ 'services', 'endpoints', 'pods' ],
+        verbs: [ 'get', 'list', 'watch' ],
+      },
+    ],
+  };
+
+local ocp_metrics_rolebinding(name, namespace, metrics_role) =
```

Comment:
@simu Copied from the rook component. Do you think it is worth creating a library in the openshift-monitoring component?

Reply:
There's a bunch of things that should be library methods in `openshift-monitoring`, this is definitely a candidate.